### PR TITLE
Link foldering

### PR DESCRIPTION
**Pull request by anastasia**
I just realized I think I meant to call this branch folder-linking. Oh well, link-foldering it is.

This pull-request links the tree folder view with the folder dropdown. By locally storing the tree node and the folder attributes (organization ID and folder ID), we can sync both parts of the app.

### CODE REVIEW

File: perma_web/static/js/links-list.js [hunk 1]
```diff
@@ -158,6 +168,56 @@ $(function() {
...
+
+    function updateLocalStorage(node) {
+      var folder = getFolderByNode(node)
+      localStorage.setItem("perma_selected_node", JSON.stringify(node));
+      localStorage.setItem("perma_selected_folder", JSON.stringify(folder));
+    }
+
+    function updatePathWithSelected(node) {
+      var nodePath = folderTree.get_path(node).join(" &gt; ");
+      var myLinks = nodePath.includes("My Links");
```
Comment (anastasia) [inline perma_web/static/js/links-list.js]: I'm assuming that there are no other folders called My Links. Faulty assumption, but might work in the meantime?

**Comment by jcushman** [comment]
I'm looking at this now -- great, fast work! Will add a few specific comments.

File: perma_web/static/js/links-list.js (hunk 1 above)
Comment (jcushman) [inline perma_web/static/js/links-list.js]: Good note -- I like the idea of something more bullet-proof. Won't node.data.organization_id be null if we're a My Links subfolder?

Comment (anastasia) [inline perma_web/static/js/links-list.js]: Yes! 
I think I just pushed a dumb test fix that will actually be solved better by your line of thought. Is My Links the only place where organization.id can be null?

**Comment by jcushman** [comment]
Great work -- pulled these changes in so they can go up on stage for people to look at. Still one or two things to iron out.

**Pull request closed without merging**